### ISSUE TITLE
Fedora 29

### ISSUE THREAD

**Issue by ct-martin** [RITlug/tigeros-repos#2]
Version number in spec file needs updating, and the key needs to be updated to the following:

```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBFvZC9QBEADHrheLRuI9z2qigVVyAH6VYUNk4kIFfpxhF7l1zqyqYfVsRBCC
7C1WliuL+i1U3G1+woPEnmdVZVZYFWepfWQyZW+v2jcn4H8MTvdir70hBq238Mam
G8ZmxqSAHHMjXeE+PPLBuzo6llJDMbeme1zUU5C5l0q/HWuIoLRXTjcn9Yvk6qEJ
qr0UnWhw5TaFl2V6MREIcjC8gtQy3ggjLHXaGUqT5gUI513jp0c/0FyUS6PLN2Ct
XjW1VyWkTltO87B79oxaWi5StpKa22RVnk2M4GTEYK6/TkU/zrhuhKBbhClEd5BT
/BrolmfOdXNLj8GEkJ9qflJ2bEV/gKn/LtqK5dUGh/GB1spSMWCN9a+FvAv2ssc2
6jTSHSWqPad/XfDP5cC9O8enw+bVjBpNM7XDsTC4kXEJzcFTkQ1nD02MHnQfxGCq
BHTgoDtuxDCb4jAZWCpX018PCOoerHeiQYnOpxE4oO/99LCrkZSY6skETOhk3TdJ
frGCj+m+1UM8uvK0M25GuAD7RaPmDr6COePZHDX/L49UgSjTaGgGSSP8cXUA1ara
3CRG1PI/QVRNlmY4zBg9N84a3hyC/t+Pw+2wbGjAy1BMpgQZCSwusCH1XfjtEEd9
pG+D7DC3ATerfwk55cWmd+v0oYyOxRotsVuU6S/Bpj6FnxmgyfEzkP5W3QARAQAB
tDZUaWdlck9TIFByb2plY3QgKExlYWQgLSBjdG1hcnRpbikgPHRpZ2Vyb3NAcml0
bHVnLmNvbT6JAj4EEwECACgFAlvZC9QCGwMFCQHhM4AGCwkIBwMCBhUIAgkKCwQW
AgMBAh4BAheAAAoJEJkZq47ouTvYJxkP+wYA48wEEUf2zwIFIbwp7TFrlkCtsFlg
U9hbCXpI37nJ9AXjD16HInSpd9jdZC7Xjd9nf3lpyPuQ+IdynjdMAF9oMP0+NZNU
vVNDiOm6P4qKtjpmqS63PcxKKDhGUwqiXoFEDZsJiixdyibOKfyZg5CCldKH+Y8E
LdhzgTfny28HbxbSOX3bs6E+VPS/DH51KyavNBFClJBaoNvtYFQ83FuQc91X6v8Z
1uE5eAKTz4L95W43FYF07s3lR85j9wdtdRWNVU/QKChCPgEiH0WWP1tZGmiqDEuJ
PQWjlF0SiTP/eHNkipzzH2qvxMLlw852pHaPS4Gwg3MlSK9YGKPWwRW4FSlyEV+H
kZlMMCijl0zjE6qFpt5kQGyEZn5o8/r9U2DvJJdhvWEhLV1SRklLGvVI7iU8E2tX
OQ1e/nMFmBrkvZx0Sn2tB+vrHEtv2GqIPDWDYAy9xfzochnCO4g86QfplUVV9dXc
yMM+G8s6H/tOhY1+5aoAMRlvIDgyua4KeD3cQ4oX3enGGySzKcthK6BQdg/jjXUS
QmKvJkIu1jVsoHN0tKfDme2y/VUwvYUsZIQnre8M8TxW/3fQaUdSjz54B2ngmsaU
5r1JHb8AbpQGzXX4+XFB7aQ5jpJDMBYcEzy0QQR0WuHbNLDT7WHjKgbM9rRztEhV
ODjm2leF+h/duQINBFvZC9QBEAC6HDj6C8ya2L9GysF27vgp0AKQgYb6G8A1931L
wZiDd0rfNuzLsuMStkzNO22rM539E8vHnn/eDdAoMJCNwNMT1Py8BZsmwrdID0F1
y2/r/z5gVeQJjjASWvY7qFW/NjxNKVPFoGAF3hyP7ydNwf+JKOPC6Ht7C0dEJm9A
UwBP2+Mx57yMVjJrhGWDlVIQnffxceGMy8eW6G7B0k0w3KdKeKEk4Hb0z49eA5XW
+HSYQwiocagb1yXUL4/g8C4nO1HoRe8V3OobcSHgTYDSV6xIut4X2YUmyPXn3Djc
5GwI3qe9noaUwylVJKE5g05mTQSfnQu6odx/4mTIeCEuDlWPagucqlPk/33FgniX
ElPD8ODjpU6B39O+3IAK/cCiJJDiauc86+EChgia1L0aw/xVnz0V8YaUc4Oz39iN
e9KaFsjeDA/5/tIy5PDFEJTwzK6rCHzPGdsLDsHjM6pd1So+KJ041layaVOdZ8gt
S1YwgqPia1xliQTTYClPfius7+iLWtKr3svvgfqREJ6q+LZHxhj+/GXBXXv82Ixe
UtAAkJNiJ3mG4OPsiZ0c0p38w0iMd9dIao4KTYlOknLuqi7t/p/kNGkRWfMiiESD
R4zO24cTdZuSBn7FYwdaoeGQxtYv5T++VkLF9KNzJKj77Q42qN/jwgBLTZO84Zyp
KV7HXwARAQABiQIlBBgBAgAPBQJb2QvUAhsMBQkB4TOAAAoJEJkZq47ouTvYNukP
/0BN5JtoXXiyKHah2SapzajU5WuU25B+NbP5AKWrXT5qw2F1CnezxMPpt7KDFEU4
GQ7wGFz1vMELkBEw8mUJUsdjNvjb4+5Y294u6Sq5N1Gnmxk74T36sWdZErls+Y0j
cP4YlqE870dgCZcjCBQ44GYVhheekz01BcT9CcJaGPtWXNMcmrp875Rn9itFiN4X
CVMx+sqvsFeplq5dVR8k9+avQ4BcjHqK5f+lB6c90//r7jAvihzHl6SVch2tJ+3B
S7nbeDJcx2Fionx19u9/cXx6gkqbNunLXLDiUUKppy4udQbqKfkZk85UA8rFsn9m
zT4ggI5AZgu2FeGnrghbxwDpk1gwUQhVjfzvMltgmSNC9l3wGtZMzRetr9KBd/rd
dFlhNvxQ0fTdBImedXqdX/VSin+ujH3RFh6dbNhtkFuCw0ie/15U3x8TvdlLCNLa
Rfxoh9Za0QJqOT7Fsq1WPAh/MUIq/E17IB8WmwocRtRcq6CL7S4ZpVNx7AC0lRak
ZBlfy2RIErrdPpucCpHSpePJSifbMOLHF2TiHR+Flu9UDBKILCMRVLORve4eSKnz
xf1JKCuAe+qYt6jMzoRgh3TVazJOv6+73tYAuirLFbVi4FaTqxgkjV+0UZp8l09z
0UX6WTJ7JE3q1BkXmKX+602Uhg0SNF46bUt2PdJ7BMau
=2yb2
-----END PGP PUBLIC KEY BLOCK-----
```

**Comment by Tjzabel** [comment]
@ct-martin done :+1:

**Comment by ct-martin** [comment]
New key (fixed):

```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBFvd+twBEADDNmOwXzmF3OZAhXOhMk4cs3Jx8bKlSURmisLUVTYKg3TUz3gV
CTQSSXAfNLVo6uAcsxWCK06fO1jUzvGMYEPi1bB3HuwJJVJxDkVnVfhMhxVBUcwJ
fGCA69m2Py2Q62/ZgvVCOf8Zp30xWsKK7gnRLDdGCK4aW7QgXT/AC1qya8+BGKl7
/0taBLDjc4g5Xjbe51PD8vUm396zlpdnosrVhR/WLDKktwdFbCtCZv7BTduXPmBz
s3/nmaVyMDnj9/pAQEintv+ZiWrsK3bKo1hk1Pllp07gCBRAsh1XtSvMEqLIiqb5
GLiSFiWc14SvKrqj/fX0T3xBgy2xstW1HNwOPYwos0XothPj4ln7SvC9nTKfDsmu
38VHjk1HLDeM/N0eFYGzIpFfvCJ2LYzJ99o9gbr6ST606ZsNqauJXqyswPxk2V1D
TXqulhTSBKx/OnFEaq+Y+5E/FbP4t9iPbRpqMJJxv6VRP5v/tzKkUJR9Hzapxo/j
b0x+7uLNHGuKMGilRJYHUtpCWkDczCy45B+4mXw9WeAV0iKbsfz/haMOXnHprlyd
AwYjtkB2dWceUUe6+bFFtzQQ+Zo7g3Utg79urPll2nCmasEGOTtApwQ8vRaQJtl+
r7rxT9qoV6esPP41tPevkTknUqEWXlcjcaDFLkU7a+yhGp1utKJIoXvkVwARAQAB
tCdUaWdlck9TIDI5IFJlbGVhc2UgPHRpZ2Vyb3NAcml0bHVnLmNvbT6JAlQEEwEI
AD4WIQTXZkmx4YvEKoPveG667VuqFJIFhAUCW9363AIbAwUJAeEzgAULCQgHAgYV
CgkICwIEFgIDAQIeAQIXgAAKCRC67VuqFJIFhPeeEACIGjk27+5OxvTb6b9HEwBz
4ySNcBmMvHtqe8xjzbJs8t+dXWfpq7pfPCguSV0/cNqgfTkjWawqcEglIceDtU43
OZkTVmSgc0p+cDnzDMttn2unmBGaddBZmqN8+FriZW6JxE+3rZ3AFcnrtlhOUM7b
Xwo2hzob8bZAxmBo6mk8HW15SiCcAFC+3unuBl6cFz4BgWf7Gwo2QR2p4HiDxdOb
Ts/U3vvZtpMyzuKm6EdiqramzCWpT7/10rhSHNtL+YD4CvJg07UZ7QSG/CX0OKyf
p3xMKWIR7PEofNdrlenpghmmqwtQ7NW4yyzpEDplApAybtugQARRj0bXRwcqA3fA
zlppr0fIj7vM+Owfup88GJpi3v72CEU7fzhmHz1Sp5C0EDuF73hiKN+Z9x0+5fYi
S8gkPZZW9uQ2i46XRL7CiLlNgJnkvEWHxSAXlckIaVSKRmZaxzX22/82BVfjHYdt
RKegI3UdUDpieNQ4RXQ05U/UydLgaeVXEujb7xJh9WT5x2Ts11LOyS/ttwH3Ejrh
WMNeK82kuUsfJtkqxZ1wi/Wa7O1ioEx09lGb+kds0SlFPAU1vLKkETIyhqe1oOTj
Ru4N5wb9kl6qUxKkLkbG8W/kY0Rt89+3c5Hh9d8MvWK9JnCOgTPbq193zMH343Nk
jEF24l3KOFEGDRrgIytuHrkCDQRb3frcARAAp8NddpU1BjMqlmopcPRgv2zcZQ3V
wMz08P/EYKdw7ecFioLToZZqDkf76wYn+043OI6v9w/lFMxNPH9Y6InO/Pu2daIk
nMYa+3T4zOQbccVdjHEEYO84Zfw3KJSRbX+NzgeEXprB0+xmmE5T0J+X8e3WoKg4
DKmEfJ1jpI0obKevIDV12z4GXd6S16TXRzf98+JClA7h+5bjks71hXxkbAZ/Q2qe
MDwghrtwE8bkRmy8yXW/IH9x6WuSpxwKqYHjDQHnnlynPDJajRaQohgGzx571krk
yvAqqoT3YEyf1hV5kxZf87oHaVp0Djyx8gxE+5ocDFRvwJpDFjT8gzvwoOMErUYb
+47YDVW6cYkHpnW2XWQILJCm5z4MKHDF9sPM0bjyR28AgsFlJV1wYsH1TH7f4GTv
3Ghc3grcFXyaNYTnRJ036nE9z9Fp+CoxgNg/yJDHede2eeARX0kd0wU6z9dWo6dE
l1gXYH+lDjhm2Xp6WkO0Hji6lHFlYL8+3IwezRerep+3/DkR7/FkhvqsjpzBQcfz
m/iAKpWW6RQLsUzsDaGDFw8+T7q/GIuMqDsBsR1tML86xVwSXwrDVHt851JohGpJ
zuaNlSanVwNshh68XiTZ6pb+GA1SyoJ6aP2SDpUjAtSqsaTrzC+1v4w/zO0+yPcc
jDH4Kqu6jQFGFSUAEQEAAYkCPAQYAQgAJhYhBNdmSbHhi8Qqg+94brrtW6oUkgWE
BQJb3frcAhsMBQkB4TOAAAoJELrtW6oUkgWEsXEP/iK0rmiESfx93EM0Ce7sN2MD
FT2LM8L6/1pAZjwrXxYWHhKOayZbKVGj7XgRGEMfm5xuTjPuw9RqSnboycxxK+Jc
9A5M9vtF13AbKR/lAAbF/A4q0obRcv5SwQTWUmwIPWwMadKAr4qYhh+Tuib4O//R
03Gq98gnAneomdISw5UedvE4w7lPg45H834RdjrVgY/E45fKBiUvG1RGkISJuk2o
chzHma1mob+Y1RXobmLlj59cxl4H3T0fCCmlq24+uFUyu9pY7l5z2meUO8HBc1Bn
oVm/lC7ohzxiltI+dBU+Q2iB9UgyfA5edpzqhUmhzMucmGY/omTts+sQKNNos5JV
GH6tXpaw5ryR0NmzuNkOnBm3SWJOAER05AOYpfuGPpD4hOwFotT+oZ4q3xr+pXls
SUBFC4BwnNQcrTY4X4KfCwSne2ggTWmrdq//WMsCJIcbBHO8leVba3iy1XvBjT4I
LRMPa2dol/KGo3aV/+RR56TuFaq5IU2HywZfNEdi/8oCDaJheLK93G8BLuFXYT3B
aOSHir/SyaIg+nYi9TP5O+5wE0cSpkhxRy75QdiKWjA4TEBEEPJczPO204G2s71B
KvE/k59WWwaSOoXr/J81RaYRxS1xvCxzgnLLgZXUoqZuyuywyHA7+AIUhrE61p+u
Hb3gDykVoP7G8GUw2/Dc
=h2LZ
```

**Comment by Tjzabel** [comment]
Fixed with f2b6228a6da94230bb31f3d125fc4719314316e4
